### ISSUE TITLE
possibility to game the Hold part of the KRE 2.0?

### ISSUE THREAD

**Issue by Polype01** [kinecosystem/rewards-engine#51]
Hello,

I checked some transactions on the KIN Explorer and have a remark regarding the hold part of the KIN KRE 2.0 

All Ecosystem Apps get some KIN based on the KIN Held on the KRE Wallet provided to Kin Foundation. (incentivizing Apps to hold KIN and not dump those on Markets)

However, are you taking into account the fact that some Apps could deposit a lot of KIN on the KRE Wallet from other wallets in order to receive weekly an increased share of KIN (for the Hold part of the KRE)?
How is the hold part of the KIN KRE equation determined?

If this has been already tackled, please disregard this and close this ticket

**Comment by TheBubblePopped** [comment]
Could you name the specific appsfor more insights please?

**Comment by Polype01** [comment]
Tapatalk for example received (officially) 2b KIN and according to the address that they use to get KRE, they own 8.9b KIN. 
So is the Hodl part of the equation based on the expected KIN received officially from the KRE (2b KIN according to my metrics) or based on 8.9b (the real amount of KIN held by Tapatalk KRE wallet)

**Comment by TheBubblePopped** [comment]
Okay so you basically [copied](https://www.reddit.com/r/KinFoundationTruth/comments/f2jnmc/top_apps_have_dumped_135b_kin_on_cointiger_while/) [kin4browns](https://www.reddit.com/r/KinFoundationTruth/comments/f4kw9l/in_true_koolaid_boi_dick_eating_fashion_polype/fhsquuc?utm_source=share&utm_medium=web2x), buiftch and VastChains [work](https://www.reddit.com/r/KinFoundationTruth/comments/f3uqr8/cointiger_top_depositors_top_10_single_deposits/). 

[Buiftch(https://www.reddit.com/r/KinFoundationTruth/comments/erxrnn/kre_payments_spreadsheet/) had some interesting insights on this. First of all you should give him credit if you use his work and second you might ask him about more insights because he had way more info than he published. 

You might also ask hiker to find out if they are trying to game the KRE and if it's working.

**Comment by Polype01** [comment]
I did not know that searching for insight on the KIN blockchain and making some comments was the privilege of a few haters. Sorry about that dude.

**Comment by TheBubblePopped** [comment]
You didn't listen. You can repack your apology and then address it to the right person (Buiftch, Browns4Kin) the next time you're gonna copy their content and repackage it as your own. 

I don't need your apology and you clearly didn't search the blockchain, you just copied their stuff, i can proof this as well but since i am more interested in hearing an answer to the question you made in your initial post from the KF i would say we can move on, right?

**Comment by Polype01** [comment]
yes indeed, you are totally off topic.

**Comment by kikengineering** [comment]
@Polype01 at a high level yes it is possible to game in the way you have mentioned but essentially we didn't feel it was that strong a game.

If an app has been paid x Kin via KRE payouts previously, as long as they have x Kin in their KRE payout (and/or confirmed storage wallet) they will receive the maximum Hold KRE payout they can. When we originally wrote the KRE 2.0 we looked at this, we discussed what you are describing, and came to the conclusion that it was "OK". The Kin could have come from somewhere else but as that amount of Kin is being held by that developer it is still being held and not sold. It doesn't really matter from a demand-on-the-market view if the KRE was originally granted via the KRE or some other means.
